app/models/pipeline_run_stage.rb

```diff
@@ -41,6 +48,19 @@ def stage_status_file(status)
     "#{pipeline_run.sample.sample_output_s3_path}/#{basename}"
   end
 
+  def dag_json_name
+    STEP_NUMBER_TO_DAG_JSON_NAMES[step_number]
+  end
+
+  def step_status_file_path
+    path_beginning = if step_number <= 2
+                       pipeline_run.sample.sample_output_s3_path
+                     else
+                       pipeline_run.sample.sample_postprocess_s3_path
+                     end
```

Comment:
is there going to be one in the `expt` folder too?

Reply:
Hm I haven't seen a sample with a `expt` folder — it seems like the experimental dag json template also outputs to the postprocess directory

Reply:
ah looks like it's been updated. guess we don't need that backwards compatibility